llvm/include/llvm/Support/raw_ostream.h

```diff
@@ -853,4 +853,6 @@ raw_ostream &operator<<(raw_ostream &OS, const std::optional<T> &O) {
 
 } // end namespace llvm
 
+bool write_console_impl(int FD, llvm::StringRef Data);
```

Comment:
This should probably be in some `llvm` namespace - not in the global one.

Reply:
Fixed by using an instance of `raw_fd_ostream` instead.